controllers/webscrapeController.js

```diff
@@ -1,21 +1,106 @@
-const axios = require("axios");
-const cheerio = require("cheerio");
-const { url } = require("./constants");
+const puppeteer = require("puppeteer");
+const { espnUrl, espnScoreUrl } = require("./constants");
 
-async function scrapeEspn(req, res) {
+async function scrapeEspn(startDate, endDate, req, res) {
```

Comment:
you are not using req in this function, needs to be removed from here and function call